flow/flow_ebos_blackoil.cpp

```diff
@@ -19,7 +19,7 @@
 // Define making clear that the simulator supports AMG
 #define FLOW_SUPPORT_AMG 1
 
-#include <opm/simulators/flow_ebos_blackoil.hpp>
+#include "flow_ebos_blackoil.hpp"
```

Comment:
Do we need to change this `#include`  convention?

Reply:
I think we should always use angle brackets and complete path (from the include root) so: no.